hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/validator/SubscriptionValidator.java

```diff
@@ -19,23 +22,27 @@
     private final MessageFilterTypeValidator messageFilterTypeValidator;
     private final TopicService topicService;
     private final SubscriptionRepository subscriptionRepository;
+    private final Optional<EndpointOwnershipValidator> endpointOwnershipValidator;
```

Comment:
To avoid playing around with `Optional`s you can add an empty implementation of `EndpointOwnershipValidator`.

Reply:
I thought about it, but I came to the conclusion that it is better to use `Optional` in this case. We do not create additional classes and unnecessary code. Empty interface and empty implementation can be misleading